.gitignore

```diff
@@ -34,3 +34,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+/test-results/
```

Comment:
I appreciate you taking the time to make sure that the repository does not include generated artifacts.